### ISSUE TITLE
Call Graphs

### ISSUE THREAD

**Issue by mvcisback** [mozilla/spiderflunky#13]
Produce the call graphs of a javascript program.

**Comment by anitsh** [comment]
I was taking a look at a lot tech for Parsers and AST. Is this project abandoned? I was just wondering if there is any further research done? I would be very much curious to know. Thank you.

**Comment by erikrose** [comment]
Yes, this project is no longer under development, nor is DXR, the larger project it was written for. Feel free to use or extend it under the terms of the Mozilla Public License, however!

**Comment by anitsh** [comment]
Thanks @erikrose.
